CIP-meta-assets (ERC20-like assets)/README.md

```diff
@@ -0,0 +1,426 @@
+---
+CIP: ?
+Title: ERC20-like assets
```

Comment:
```suggestion
Title: Programmable token-like assets
```
Of course @michele-nuzzi as author, or other community members, are welcome to provide alternative names... this is just a pilot to see how it looks everywhere.